app-insights.tf

```diff
@@ -1,8 +1,16 @@
-resource "azurerm_application_insights" "apim" {
-  name                = local.name
-  location            = var.location
-  resource_group_name = var.virtual_network_resource_group
-  application_type    = "other"
-  retention_in_days   = 30
-  tags                = var.common_tags
+module "application_insights" {
+  source = "git@github.com:hmcts/terraform-module-application-insights?ref=main"
+
+  env     = var.env
+  product = var.product
+  name    = "${var.product}-${var.component}-appinsights"
```

Comment:
think name should be this
`${var.department}-api-mgmt`

Reply:
if you se on the original code, it was `local.name`

Reply:
could we just use the local.name for the name or will that cause problems?

Reply:
nope- `local.name` also got `env` in it so it will have incorrect name